service/logging.go

```diff
@@ -25,7 +25,7 @@ type logmw struct {
 func (mw logmw) RequestEnrollment(ctx context.Context, enrollSecret, hostIdentifier string) (errcode string, reauth bool, err error) {
 	defer func(begin time.Time) {
 		uuid, _ := uuid.FromContext(ctx)
-		mw.logger.Log(
+		mw.logger.Debug(
```

Comment:
These logging middleware are also used by the server and should not be enforcing a level. The level was enforced when the logger is created. 

See the pull request that removed `Debug` for more info https://github.com/kolide/launcher/pull/181